binjastub/requirements.txt

```diff
@@ -0,0 +1,4 @@
+git+git://github.com/trailofbits/manticore.git@a16a01148bde965803cc3a2101bf4e03b0dfe340
+pygments>=2.7.0,<2.9.0
+crytic-compile>=0.2.0
+capstone @ git+https://github.com/aquynh/capstone.git@1766485c0c32419e9a17d6ad31f9e218ef4f018f#subdirectory=bindings/python
```

Comment:
Would be nice to only have a single `requirements.txt`. This looks very similar to the top-level one

Reply:
Yeah. My understanding is that if we use a subdirectory (`binjastub` in our case), binary ninja plugin mgr parses the requirements.txt in our subdirectory, hence this extra requirements.txt.

```python
# https://github.com/Vector35/community-plugins/blob/master/generate_index.py#L127
requirements_txt = ""
try:
    if "subdir" in plugin:
        req_json = getfile(f"{projectUrl}/contents/{plugin['subdir']}/requirements.txt?ref={plugin['tag']}").json()
    else:
        req_json = getfile(f"{projectUrl}/contents/requirements.txt?ref={plugin['tag']}").json()
```

Not sure how we should best merge these 2 requirements.txt files (maybe a symlink?)

Reply:
I think the file in `binjastub` is required in order for the plugin manager to work, but we might be able to delete the top-level `requirements.txt` and adjust the path on this line to point to `binjastub/requirements.txt` https://github.com/trailofbits/ManticoreUI/blob/70fd9e51454083eb90940fa8e5668d38b0f8d743/Makefile#L4